tests/cibyl/e2e/test_config.py

```diff
@@ -45,7 +45,6 @@ class TestSchema(EndToEndTest):
     """Tests configuration files against their schema.
     """
 
-    @skip(reason='Test fails to a reason different to what is tested here')
```

Comment:
skip is removed but still imported, flake is complaining in CI about it

Reply:
True

Reply:
Fixed